### DISCUSSION TITLE
如何通过js使访问地址参数变化，但页面不刷新

### DISCUSSION THREAD

**Discussion by qinghechaoge** [vbenjs/vue-vben-admin#2746] [General]
类似于有道云笔记，打开文件夹，顶部地址变了，但页面没有刷新

**Comment by ghost** [comment]
哈希路由？

**Reply by qinghechaoge** [reply]
对，vben支持吗 怎么用

**Reply by ghost** [reply]
好像vben默认就是哈希路由。。。
你说的情况应该是多个页面的整体布局没变，但是局部数据发生变化，一般是在配置路由的时候指定一下对应的布局就可以了

**Reply by qinghechaoge** [reply]
好像把不太对，我觉得要用路由守卫，但是我控制不了不让页面重新加载

> 好像vben默认就是哈希路由。。。 你说的情况应该是多个页面的整体布局没变，但是局部数据发生变化，一般是在配置路由的时候指定一下对应的布局就可以了

**Reply by ghost** [reply]
那就不太清楚了，我也是刚刚开始看这个项目的新手，抱歉
